proto/tendermint/p2p/types.proto

```diff
@@ -32,6 +32,7 @@ message PeerInfo {
   string                    id             = 1 [(gogoproto.customname) = "ID"];
   repeated PeerAddressInfo  address_info   = 2;
   google.protobuf.Timestamp last_connected = 3 [(gogoproto.stdtime) = true];
+  bool                      inactive       = 4;
```

Comment:
Sort of a nit, but the name `inactive` implies that it could in principle become active again. I think based on how we use this, something like `blocked` or `excluded` might be more descriptive?

Reply:
It is possible that this node be marked as active again if they successfully dial us

Reply:
I believe this type is only used by tendermint to persist peer info to the database. Pex just gossips addresses without much extra information (alas, perhaps)

Reply:
Or, incompatible?